src/universal/modules/outcomeCard/components/OutcomeCardTextarea/OutcomeCardTextarea.js

```diff
@@ -33,66 +42,97 @@ class OutcomeCardTextArea extends Component {
     }
   }
 
-  render() {
+  setEditing = () => {
+    this.setState({isEditing: true});
+  };
+
+  unsetEditing = () => {
+    this.setState({isEditing: false});
+  }
+
+  renderEditing() {
     const {
-      cardHasHover,
       handleSubmit,
       input,
       isActionListItem,
       isArchived,
-      isProject,
-      doFocus,
-      styles,
+      styles
     } = this.props;
-
     const contentStyles = css(
       !isActionListItem && styles.content,
       isActionListItem && styles.actionListContent,
-      isProject && !isArchived && cardHasHover && styles.contentWhenCardHovered,
       isArchived && styles.isArchived,
-      !isProject && cardHasHover && styles.actionContentWhenCardHovered,
-      !isProject && styles.descriptionAction
     );
 
-    let textAreaRef;
     const handleBlur = () => {
       const {input: {value}} = this.props;
       if (value) {
         // if there's no value, then the document event listener will handle this
         input.onBlur();
+        this.unsetEditing();
         handleSubmit();
       }
     };
-    const handleKeyPress = () => {
-      // TODO fix me there's a little lag here
-      // handleSubmit();
-    };
-    const setRef = (c) => {
-      textAreaRef = c;
-    };
-    const submitOnEnter = (e) => {
-      // hitting enter (not shift+enter) submits the textarea
-      if (e.key === 'Enter' && !e.shiftKey) {
-        textAreaRef.blur();
-      }
-    };
-
+    const shouldAutoFocus = true;
     return (
       <Textarea
         {...input}
-        ref={setRef}
         className={contentStyles}
         disabled={isArchived}
         maxLength="255"
         placeholder="Type your outcome here"
         onBlur={handleBlur}
         onDrop={null}
-        onKeyDown={submitOnEnter}
-        onKeyUp={handleKeyPress}
-        autoFocus={doFocus}
+        autoFocus={shouldAutoFocus}
       />
     );
   }
+
+  renderMarkdown() {
+    const {
+      styles,
+      cardHasHover,
+      isProject,
+      isActionListItem,
+      isArchived,
+      input: {value}
+    } = this.props;
+    const markdownStyles = css(
+      styles.markdown,
+      !isActionListItem && styles.content,
+      isActionListItem && styles.actionListContent,
+      isProject && !isArchived && cardHasHover && styles.contentWhenCardHovered,
+      !isProject && cardHasHover && styles.actionContentWhenCardHovered,
+      !isProject && styles.descriptionAction
+    );
+    const markdownCustomComponents = {
+      Link: LinkNewTab
+    };
+
+    return (
+      <div
+        onClick={!isArchived && this.setEditing}
+        className={markdownStyles}
+      >
+        <ReactMarkdown
```

Comment:
I've noticed that when the user enters multiple line-breaks (i.e. `\n\n`) they aren't rendered as `<br>` tags. I'm not sure how I feel about this yet. I think we should test it with users (vs. going too far in trying to change the behavior). But, I want to flag it.